www/includes/easyparliament/templates/html/mp/divisions.php

```diff
@@ -0,0 +1,206 @@
+    <div class="<?= $current_assembly ?>">
```

Comment:
Should this be shared with profile and votes? (Definitely yes, because that would prevent #795 happening again :) )